docs/en/integrations/data-visualization/astrato-and-clickhouse.md

```diff
@@ -0,0 +1,105 @@
+
+- Select **Data** in the sidebar, and select the **Data Connection** tab 
+(or, navigate to this link: https://app.astrato.io/data/sources)
+​
+- Click on the **New Data Connection** button in the top right side of the screen.
+<img  src={require('./images/astrato_1_dataconnection.png').default}  class="image"  alt="Astrato Data Connection"  style={{max-width:'50%',  'background-color':  'transparent'}}/>
+
+<br/>
+
+- Select **ClickHouse**. 
+<img  src={require('./images/astrato_2a_clickhouse_connection.png').default}  class="image"  alt="Astrato ClickHouse Data Connection"  style={{max-width:'50%',  'background-color':  'transparent'}}/>
+- Complete the required fields in the connection dialogue box 
+
+<img  src={require('./images/astrato_2b_clickhouse_connection.png').default}  class="image"  alt="Astrato connect to ClickHouse required fields"  style={{width:'50%',  'background-color':  'transparent'}}/>
+
+- Click **Test Connection**.
+    
+- If the connection is successful, give the data connection a **name** and click **Next.**
+
+- Set the **user access** to the data connection and click **connect.**  
+​
+<img  src={require('./images/astrato_2b_clickhouse_connection.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
+
+-   A connection is created and a dataview is created.
+    
+
+ℹ️  **Please note**: if a duplicate is created, a timestamp is added to the data source name.
+
+---
+
+# Creating a semantic model / Data View in Astrato, with ClickHouse data
+
+In our Data View editor, you will see all of your Tables and Schemas in ClickHouse, select some to get started.
+
+<img  src={require('./images/astrato_4a_clickhouse_data_view.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
+
+
+Now that you have your data selected, go to define the **data view**. Click define on the top right of the webpage.
+
+In here, you are able to join data, as well as, **create governed dimensions and measures** - ideal for driving consistency in business logic across various teams. 
+
+<img  src={require('./images/astrato_4b_clickhouse_data_view.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
+
+
+**Astrato intelligently suggests joins** using your meta data, including leveraging the keys in ClickHouse. Our suggested joins make it easy for you to gets started, working from your well-governed ClickHouse data, without reinventing the wheel. We also show you **join quality** so that you have the option to review all suggestions, in detail, from Astrato.
+
+<img  src={require('./images/astrato_4c_clickhouse_data_view.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
```

Comment:
file does not exist.

should be: `docs/en/integrations/data-visualization/images/astrato_4c_clickhouse_completed_data_view.png`